packages/wallet/src/types.ts

```diff
@@ -82,6 +82,7 @@ export interface ObservableWallet {
   readonly protocolParameters$: Observable<Cardano.ProtocolParameters>;
   readonly addresses$: Observable<GroupedAddress[]>;
   readonly assets$: Observable<Assets>;
+  readonly fatalError$: Observable<unknown>;
```

Comment:
A doc comment on this would be nice (explaining it's a catch-all and that wallet is probably 'bricked' when this emits)